src/common/types.hpp

```diff
@@ -114,12 +112,12 @@ class Ip6Address
     }
 
     /**
-     * Constructor with an Thread Ip6 address.
+     * Constructor with an Ip6 address.
      *
-     * @param[in]   aAddress    The Thread Ip6 address.
+     * @param[in]   aAddress    The Ip6 address.
      *
      */
-    Ip6Address(const otIp6Address &aAddress);
```

Comment:
Is it possible to use forward declaration to implement `Ip6Address(const otIp6Address &aAddress);` without including `openthread/ip6.h` ?

Reply:
I think it is possible. But since this is blocking our sync. I suggest we figure out a better solution later.